src/module/actor/sheets/SR5BaseActorSheet.ts

```diff
@@ -163,62 +173,128 @@ export class SR5BaseActorSheet extends foundry.appv1.sheets.ActorSheet {
      *
      * This includes dropping them onto this actor.
      */
-    getForbiddenItemTypes(): string[] {
+    getForbiddenItemTypes(): Item.ConfiguredSubType[] {
         return [];
     }
 
     /**
      * Extend and override the default options used by the 5e Actor Sheet
      * @returns {Object}
      */
-    static override get defaultOptions() {
-        return foundry.utils.mergeObject(super.defaultOptions, {
-            classes: ['sr5', 'sheet', 'actor'],
-            width: 930,
-            height: 690,
+    static override DEFAULT_OPTIONS: any = {
```

Comment:
It's failing because the functions `addEffect` and `deleteEffect` are return other value than void, if you fix it, the type is fixed as well